lib/products/crm/crm-module.coffee

```diff
@@ -80,10 +80,17 @@ class CrmModule extends BaseModule
             _.each(v, (_v) =>
               _.extend(result[k],@processRecord(_v))
             )
+          else if _.has(v, 'code')
+            infoKeys = ['code', 'message']
+            result[k] = {}
+            for i, key of infoKeys
+              result[k][key] = v[key][0]
       else if _.has(record,'_') and _.has(record,'$') and _.has(record.$,'param')
         result[record.$.param] = record._
       else if _.has(record,'_') and _.has(record,'$') and _.has(record.$,'val')
         result[record.$.val] = record._
+      else if _.has(record,'response')
```

Comment:
I am wondering about this. 

Because we are parsing the <response> for all other records including the ones with <error>, why we need to specify 'response' here ? wouldn't  success also be treated like the error response?

Sorry this function `processRecord ` start to really scare me 😨

Reply:
Response from Zoho contains `success` field inside `response`.

If we omit this we'll get empty response body.

If you wanna avoid this we can add this code
```
processed = @processRecord(response.data.response)
response.data = processed
```